### PR TITLE
op-guide: fix incorrect IP address parsing and other inline code formatting

### PR DESCRIPTION
**Pull request by dcalvin**
Using escapes for IP addresses can cause unintentional parsing of IP type values, as shown in 
https://pingcap.com/docs-cn/op-guide/pd-configuration/#pd-%E9%85%8D%E7%BD%AE%E5%8F%82%E6%95%B0 

This fix also updates some punctuations.

### CODE REVIEW

File: op-guide/pd-configuration.md [hunk 1]
```diff
@@ -9,76 +9,76 @@ PD 可以通过命令行参数或环境变量配置。
...
-+ 例如，docker 内部 IP 地址为 172.17.0.1，而宿主机的 IP 地址为 192.168.100.113 并且设置了端口映射 -p 2380:2380，那么可以设置为 \-\-advertise-peer-urls="http://192.168.100.113:2380"，其他 PD 节点可以通过 http://192.168.100.113:2380 来找到这个服务
++ 例如，docker 内部 IP 地址为 172.17.0.1，而宿主机的 IP 地址为 192.168.100.113 并且设置了端口映射 `-p 2380:2380`，那么可以设置为 `--advertise-peer-urls="http://192.168.100.113:2380"`，其他 PD 节点可以通过 `http://192.168.100.113:2380` 来找到这个服务。
 
 ## `--client-urls`
 
-+ 处理客户端请求监听 URL 列表
-+ 默认："http://127.0.0.1:2379"
-+ 如果部署一个集群，\-\-client-urls 必须指定当前主机的 IP 地址，例如 "http://192.168.100.113:2379"，如果是运行在 docker 则需要指定为 "http://0.0.0.0:2379"
++ 处理客户端请求监听 URL 列表。
++ 默认："`http://127.0.0.1:2379`"
```
Comment (CaitinChen) [inline op-guide/pd-configuration.md]:
```suggestion
+ 默认：`http://127.0.0.1:2379`
```

File: op-guide/pd-configuration.md [hunk 2]
```diff
@@ -9,76 +9,76 @@ PD 可以通过命令行参数或环境变量配置。
...
++ 如果部署一个集群，\-\-client-urls 必须指定当前主机的 IP 地址，例如`http://192.168.100.113:2379"`，如果是运行在 docker 则需要指定为 `http://0.0.0.0:2379`。
```
Comment (CaitinChen) [inline op-guide/pd-configuration.md]:
```suggestion
+ 如果部署一个集群，\-\-client-urls 必须指定当前主机的 IP 地址，例如 `http://192.168.100.113:2379"`，如果是运行在 docker 则需要指定为 `http://0.0.0.0:2379`。
```

File: op-guide/pd-configuration.md [hunk 3]
```diff
@@ -9,76 +9,76 @@ PD 可以通过命令行参数或环境变量配置。
...
 
 ## `--peer-urls`
 
 + 处理其他 PD 节点请求监听 URL 列表。
-+ default: "http://127.0.0.1:2380"
-+ 如果部署一个集群，\-\-peer-urls 必须指定当前主机的 IP 地址，例如 "http://192.168.100.113:2380"，如果是运行在 docker 则需要指定为 "http://0.0.0.0:2380"
++ default: "`http://127.0.0.1:2380`"
```
Comment (CaitinChen) [inline op-guide/pd-configuration.md]:
```suggestion
+ default: `http://127.0.0.1:2380`
```

File: op-guide/pd-configuration.md [hunk 4]
```diff
@@ -9,76 +9,76 @@ PD 可以通过命令行参数或环境变量配置。
...
++ 如果部署一个集群，\-\-peer-urls 必须指定当前主机的 IP 地址，例如 `"http://192.168.100.113:2380"`，如果是运行在 docker 则需要指定为 `"http://0.0.0.0:2380"`。
```
Comment (CaitinChen) [inline op-guide/pd-configuration.md]:
```suggestion
+ 如果部署一个集群，\-\-peer-urls 必须指定当前主机的 IP 地址，例如 `http://192.168.100.113:2380`，如果是运行在 docker 则需要指定为 `http://0.0.0.0:2380`。
```

File: op-guide/pd-configuration.md [hunk 5]
```diff
@@ -9,76 +9,76 @@ PD 可以通过命令行参数或环境变量配置。
...
 + 默认：""
-+ 如果你指定了配置文件，PD 会首先读取配置文件的配置。然后如果对应的配置在命令行参数里面也存在，PD 就会使用命令行参数的配置来覆盖配置文件里面的
++ 如果你指定了配置文件，PD 会首先读取配置文件的配置。然后如果对应的配置在命令行参数里面也存在，PD 就会使用命令行参数的配置来覆盖配置文件里面的。
 
 ## `--data-dir`
 
-+ PD 存储数据路径
-+ 默认："default.${name}"
++ PD 存储数据路径。
++ 默认："`default.${name}`"
```
Comment (CaitinChen) [inline op-guide/pd-configuration.md]:
```suggestion
+ 默认：`default.${name}`
```

File: op-guide/pd-configuration.md [hunk 6]
```diff
@@ -9,76 +9,76 @@ PD 可以通过命令行参数或环境变量配置。
...
 
 ## `--initial-cluster`
 
 + 初始化 PD 集群配置。
 + 默认："{name}=http://{advertise-peer-url}"
-+ 例如，如果 name 是 "pd", 并且 `advertise-peer-urls` 是 "http://192.168.100.113:2380", 那么 `initial-cluster` 就是 pd=http://192.168.100.113:2380
++ 例如，如果 name 是 "pd", 并且 `advertise-peer-urls` 是 `"http://192.168.100.113:2380"`, 那么 `initial-cluster` 就是 `pd=http://192.168.100.113:2380`。
```
Comment (CaitinChen) [inline op-guide/pd-configuration.md]:
```suggestion
+ 例如，如果 name 是 "pd", 并且 `advertise-peer-urls` 是 `http://192.168.100.113:2380`, 那么 `initial-cluster` 就是 `pd=http://192.168.100.113:2380`。
```

**Comment by dcalvin** [comment]
Thanks @CaitinChen  PTAL again.

**Pull request merged**